File: app/src/main/java/com/seafile/seadroid2/framework/model/ContextModel.java
package com.seafile.seadroid2.framework.model;

public class ContextModel {
    //account
    public String email;
    public String server;
    public String name;


    //repo
    public String repo_id;
    public String repo_name;

    public String type;   // account/repo/dirent

    /**
     * parent_dir + name
     */
    public String full_path;
    public String permission;
    public boolean encrypted;
}

File: app/src/main/java/com/seafile/seadroid2/framework/model/ContextModel.kt
package com.seafile.seadroid2.framework.model

class ContextModel {
    @JvmField
    var repo_id: String? = null

    @JvmField
    var repo_name: String? = null

    @JvmField
    var type: String? = null

    /**
     * parent_dir + name
     */
    @JvmField
    var full_path: String? = null

    @JvmField
    var permission: String? = null

    @JvmField
    var encrypted: Boolean = false
}
